src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -299,3 +302,80 @@ export const MyHighlightsImage = styled.img`
   height: ${myHighlightsImageHeight}rem;
   margin-top: ${popupBodyPadding}rem;
 `;
+
```

Comment:
this file is getting big, can we separate the empty state stuff from the highlights stuff?